backend/src/main/java/com/google/minicrm/data/Lead.java

```diff
@@ -153,9 +167,39 @@ public Entity asEntity() {
     leadEntity.setProperty("creativeId", creativeId);
     leadEntity.setProperty("notes", notes);
     leadEntity.setProperty("status", status.ordinal());
```

Comment:
prefer not to use ordinal(). You should define an actual method such as getIndex() in your enum to get the numeric value.

Reply:
will fix.

Reply:
Since we are using getIndex(), should we define another method like getFromIndex(int index) to get the enum at the certain index as well?
Currently we are using LeadStatus.values().
this.status = LeadStatus.values()[((Long) entity.getProperty("status")).intValue()];

Reply:
that sounds like a good idea!

Reply:
will do!